requirements.txt

```diff
@@ -1,2 +1,2 @@
-elasticsearch==1.9.0
-mock==1.0.1
\ No newline at end of file
+elasticsearch==6.3.0
```

Comment:
Wow, that's a huge version jump.  Looks like the old elasticmock is just really out of date?

Reply:
yes, so probably more changes will be needed, and an upstream PR probably won't be accepted until coverage is complete, but for now I'm just hacking the parts we need to use